Tests/SentryTests/Integrations/Performance/IO/DataSentryTracingIntegrationTests.swift

```diff
@@ -87,6 +102,25 @@ class DataSentryTracingIntegrationTests: XCTestCase {
             // URL to a file that is not a file but should exist at all times
             URL(string: "https://raw.githubusercontent.com/getsentry/sentry-cocoa/refs/heads/main/.gitignore")!
         }
+
+        func tearDown() throws {
+            clearTestState()
+
+            // Delete files created by the test run
+            let manager = FileManager.default
+            if fileUrlToRead != nil && manager.fileExists(atPath: fileUrlToRead.path) {
+                try manager.removeItem(at: fileUrlToRead)
+            }
+            if fileUrlToWrite != nil && manager.fileExists(atPath: fileUrlToWrite.path) {
+                try manager.removeItem(at: fileUrlToWrite)
+            }
+            if ignoredFileUrlToRead != nil && manager.fileExists(atPath: ignoredFileUrlToRead.path) {
+                try manager.removeItem(at: ignoredFileUrlToRead)
+            }
+            if ignoredFileUrlToWrite != nil && manager.fileExists(atPath: ignoredFileUrlToWrite.path) {
+                try manager.removeItem(at: ignoredFileUrlToWrite)
+            }
```

Comment:
`m`: Why do you check if the file exists before removing it? Can't we simply just remove it? That would be less code.

Reply:
Because it throws with "file not found"

Reply:
Oh my 🤦. It's late sorry. I thought it would simply return false. Ignore me.